src/terminal/adapter/InteractDispatch.cpp

```diff
@@ -248,8 +248,5 @@ bool InteractDispatch::FocusChanged(const bool focused) const
     }
     // Does nothing outside of ConPTY. If there's a real HWND, then the HWND is solely in charge.
 
-    // Theoretically, this could be propagated as a focus event as well, to the
-    // input buffer. That should be considered when implementing GH#11682.
-
     return true;

```

Comment:
we should make sure that a VT Input client on the far receiving end doesn't get _two events each focus change_. I was wondering if it would because:

1. we might pass one through (original one)
2. we add a `FOCUS_EVENT` to the input buffer, which TerminalInput later converts into another focus VT sequence

Reply:
WHOOPS IT DOES THE OPPOSITE.

Terminal doesn't send _any_ focus events to the client. It sets the terminal into focus mode, it sends the focus in, `InteractDispatch` then enables CONSOLE_FOREGROUND, and then returns _and never writes the focus event to the buffer._

Reply:
_oops_!

Reply:
fixed it

![gh-11682-in-terminal](https://user-images.githubusercontent.com/18356694/164087915-cac538fa-5913-47d2-8eab-1ee32891e522.gif)